module/plugins/problems/problems.py

```diff
@@ -35,7 +35,7 @@
 
 
 def get_page():
-    app.bottle.redirect("/all?search=%s" % DEFAULT_FILTER)
+    app.bottle.redirect("/all?search=bi:>=%d %s" % (app.problems_business_impact, DEFAULT_FILTER))
```

Comment:
Should be something like `max(app.problems_ business_impact, user.min_business_impact)` or maybe `user.min_business_impact or app.problems_business_impact` (which allows user to set less than app.problems_business_impact).

And the bi filter should be added at the end of the search, not at begining.

Reply:
Ok with this using the maximum of both BI and setting the filter on search end. I give a try with this to confirm it is consistent

Reply:
1. Setting the BI filter in the last position is ok to me. Only two points: it is not the most effective because the BI filtering is the last filter to get applied ... and on last position the `bi>=`filter is not always visible on small screens :/

1. Using `user.min_business_impact or app.problems_business_impact` is a very good and pythonic solution :wink:

Reply:
Effectiveness depends on the search. On default problems search, isnot:OK will be way more discriment (then effective) than bi:>1.

And I'm fine with bi filter not always visible on small screens because it is the least important filter.